settings/DevHome.Settings/Strings/en-us/Resources.resw

```diff
@@ -406,4 +465,13 @@
   <data name="Settings_Accounts_NoProvidersContentDialog_SecondaryButtonText" xml:space="preserve">
     <value>Cancel</value>
   </data>
-</root>
+  <data name="Settings_ExperimentalFeatures_Description" xml:space="preserve">
+    <value>Configure experimental features</value>

```

Comment:
Nit: technically these should have `<comment>`s as well. Obviously not all resw files in Dev Home do, but I assume someone will complain eventually.